### PR TITLE
Romanian translations

### PR DESCRIPTION
**Pull request by fabsb**
This PR updates faulty romanian translations and adds missing ones.

### CODE REVIEW

**Comment by jubalh** [comment]
Pretty cool, thanks @fabsb  :)
Can I merge is there need for further translation? (Since the PR is named WIP)

**Comment by fabsb** [comment]
I would like to check the romanian translations for the other modules as I noticed some missing ones. I assume the PR will be ready for merge tonight.

**Comment by jubalh** [comment]
Alright then

File: plugin-networkmonitor/translations/networkmonitor_ro_RO.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@
     <message>
         <location filename="../lxqtnetworkmonitorconfiguration.ui" line="14"/>
         <source>Network Monitor settings</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Setări monitorizare rețea</translation>
```
Comment (fabsb) [inline plugin-networkmonitor/translations/networkmonitor_ro_RO.ts]: @jubalh Should I remove the *unfinished* type attribute after updating translations or will those attributes be updated in a later pass?

Comment (palinek) [inline plugin-networkmonitor/translations/networkmonitor_ro_RO.ts]: > Should I remove the unfinished type attribute after updating translations 

Yes, please

**Pull request merged**